Makefile

```diff
@@ -47,7 +47,7 @@ border-radius-styles:
 	@echo "Generating border-radius style definitions..."
 	@node genBorderRadius.js
 
-LINT_MAX_LESS_PROBLEMS := 125
+LINT_MAX_LESS_PROBLEMS := 322
```

Comment:
This seems like a lot of added problems...

![bitmoji](https://render.bitstrips.com/v2/cpanel/10219132-255968629_2-s4-v1.png?transparent=1&palette=1&width=246)